scripts/increase-test-logging.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Yes.  They are not at all generalized, but will be.  In the meantime, a starting point to build other scripts?